README.md

```diff
@@ -1,31 +1,554 @@
-#### ⚠️ DISCLAIMER
+# identity-server
 
-Use these artefacts as a reference to build your deployment artefacts. Existing artefacts only developed to demonstrate a reference deployment and should not be used as is in production
+A Helm chart for WSO2 Identity server. This Helm chart can be used to deploy highly available and scalable WSO2 identity server deployment.
 
-------------------------------------------------------------------
+## Required permission
+User or service principle who installs the Helm chart, needs to possess actions `"create", "get", "list", "update", "delete"` on following K8s kinds,
 
-# Kubernetes and Helm Resources for WSO2 Identity And Access Management
+| Kind                    | API Version                      |
+|-------------------------|----------------------------------|
+| ConfigMap               | v1                               |
+| Deployment              | apps/v1                          |
+| HorizontalPodAutoscaler | autoscaling/v1                   |
+| Ingress                 | networking.k8s.io/v1             |
+| PersistentVolume        | v1                               |
+| PersistentVolumeClaim   | v1                               |
+| PodDisruptionBudget     | policy/v1                        |
+| Role                    | rbac.authorization.k8s.io/v1     |
+| RoleBinding             | rbac.authorization.k8s.io/v1     |
+| SecretProviderClass     | secrets-store.csi.x-k8s.io/v1    |
+| Service                 | v1                               |
+| ServiceAccount          | v1                               |
+| Secret                  | v1                               |
 
-*Kubernetes and Helm Resources for container-based deployments of WSO2 Identity Server deployment patterns.*
 
-* A clustered deployment of WSO2 Identity Server
+# Quick Start Guide
 
-## Deploy Kubernetes resources
+## Prerequisites
+* Kubernetes ingress controller. Default integration is [Kubernetes Nginx ingress controller](https://github.com/kubernetes/ingress-nginx).
+* An active [WSO2 Subscription](https://wso2.com/subscription).
 
-In order to deploy Kubernetes resources for each deployment pattern, follow the **Quick Start Guide** for each deployment pattern
-given below:
 
-### Simple
+1. Add the WSO2 Helm chart repository.
```

Comment:
Shall we add a heading for this section.  This section confused with the Prerequisites.